services/brig/src/Brig/API/Public.hs

```diff
@@ -1444,10 +1444,10 @@ getHandleInfo self handle = do
         Just ownerId -> do
           ownerProfile <- lift $ API.lookupProfile self (Qualified ownerId domain)
           owner <- filterHandleResults self (maybeToList ownerProfile)
-          return $ Public.UserHandleInfo . Public.profileQualifiedId <$> listToMaybe owner
+          return $ listToMaybe owner
     getRemoteHandleInfo = do
-      Log.info $ (Log.msg $ Log.val "getHandleInfo - remote lookup") Log.~~ Log.field "domain" (show (qDomain handle))
-      Federation.getUserHandleInfo handle
+      Log.info $ Log.msg (Log.val "getHandleInfo - remote lookup") Log.~~ Log.field "domain" (show (qDomain handle))
+      undefined $ Federation.getUserHandleInfo handle
```

Comment:
I see an undefined in a PR not marked as draft.

Reply:
Hmm, some test is missing!

Reply:
There is a bug in pipeline! The tests are fine, phew!